### DISCUSSION TITLE
Intergrate with Ragflow

### DISCUSSION THREAD

**Discussion by chuwe000** [langfuse/langfuse#3755] [Ideas]
### Describe the feature or potential improvement

Hi 
We are tring use Ragflow to build our AI chatbot. Is it possible that use langflow to trace those prompt event? just like langfuse integrate with flowise or langflow?

### Additional information

Ref. [Ragflow](https://github.com/infiniflow/ragflow)

**Comment by marcklingen** [comment]
thanks for sharing @chuwe000!

**Comment by jannikmaierhoefer** [comment]
This is avaliable and documented here: https://langfuse.com/docs/integrations/ragflow
